test/valid-data/type-mapped-annotated-string/main.ts

```diff
@@ -8,3 +10,8 @@ type CurrencyISO = string;
  * Exchange rate
  */
 export type ExchangeRate = Record<CurrencyISO, number>;
+
+/**
+ * Size chart
+ */
+export type SizeChart = Record<Size, number>;
```

Comment:
I left the original 'not-imported' `CurrencyISO` type as-is, and added a new `imported` `SizeChart` type to test the issue